app/components/card-hazard.tsx

```diff
@@ -79,12 +79,12 @@ const CardHazard: React.FC<CardHazardProps> = ({
         <PopoverTrigger>
           <VStack cursor={"pointer"} alignItems={"flex-start"} h={"100%"}>
             <CardHeader p={0}>
-              <Text textStyle="textBig" fontWeight={"700"}>
+              <Text textStyle="cardTitle" fontWeight={"700"}>
```

Comment:
this is to separate out the blue-colored card title from normal `textBig` with color `grey/900`